src/java/org/apache/cassandra/db/compaction/LeveledGenerations.java

```diff
@@ -74,7 +75,7 @@
 
     Set<SSTableReader> get(int level)
     {
-        if (level > levelCount() - 1 || level < 0)
+        if (level >= levelCount() || level < 0)
```

Comment:
This and below changes in this file are purely cosmetic. I can revert if needed, however, I feel they make code clearer.